crates/cosmos/cosmos-integration-tests/tests/bootstrap_legacy.rs

```diff
@@ -1,13 +1,23 @@
 use std::sync::Arc;
 
+use hermes_cosmos_chain_components::traits::eip::eip_type::EipQueryType;
 use hermes_cosmos_integration_tests::contexts::bootstrap_legacy::LegacyCosmosBootstrap;
 use hermes_cosmos_integration_tests::init::init_test_runtime;
 use hermes_cosmos_relayer::contexts::build::CosmosBuilder;
+use hermes_cosmos_test_components::types::dynamic_gas_config::DynamicGasConfig;
 use hermes_error::types::Error;
 use hermes_test_components::bootstrap::traits::chain::CanBootstrapChain;
 
 #[test]
 fn test_cosmos_legacy_bootstrap() -> Result<(), Error> {
+    let maybe_dynamic_gas_fee_config = std::env::var("DYNAMIC_GAS_MULTIPLIER")
```

Comment:
We don't need to have dynamic gas configurable in every test. Let's just enable it by default in tests with modern chains, and disable it in legacy tests. We should also implement `Default` for `DynamicGasConfig`, so that we don't need to specify it in all places.

Reply:
Default defined in 8b636cc51919eb3d6584f7f87bedc9f31bef91aa

Should we allow configuration of the multiplier or max?